fpmms/tools.py

```diff
@@ -606,14 +633,22 @@ def pipeline_step(
         with ThreadPoolExecutor(max_workers=NUM_WORKERS) as executor:
             futures = []
             for i in range(0, len(parsed), 1000):
-                futures.append(executor.submit(get_contents, session, parsed[i:i+1000], event_name))
+                futures.append(
+                    executor.submit(
+                        get_contents, session, parsed[i : i + 1000], event_name
```

Comment:
Maybe rename the batch size constant and use it here too?

Reply:
sure